statdivlab/happi
# Address issue #50

I made changes to the vignette to address Issue #50 and added other useful comments/code for happi's usage:

- how to use happi to test many genes
- how to conduct FDR correction using BY
- how to use epsilon to conduct sensitivity analyses.

I'd appreciate another set of eyes on the vignette and welcome any recommendations!

No other files should be changed aside from the vignette.
